# C. jejuni Pangenome & Genomics Literature

> **Dataset**: 493 strains, ~96 BioProjects | **Taxon ID**: 197 | **Reference**: NCTC 11168
>
> Curated literature supporting the pyphylon *Campylobacter jejuni* pangenome analysis pipeline.

---

## 1. Pangenome Studies

- **Zhong et al. (2022)** Pan-Genome Analysis of Campylobacter: Insights on the Genomic Diversity and Virulence Profile. *Microbiology Spectrum*, 10(5):e01029-22. DOI: [10.1128/spectrum.01029-22](https://doi.org/10.1128/spectrum.01029-22) | PubMed: [36069574](https://pubmed.ncbi.nlm.nih.gov/36069574/) | [PDF](papers/Zhong_2022.pdf)

- **Méric et al. (2014)** A Reference Pan-Genome Approach to Comparative Bacterial Genomics: Identification of Novel Epidemiological Markers in Pathogenic Campylobacter. *PLOS ONE*, 9(3):e92798. DOI: [10.1371/journal.pone.0092798](https://doi.org/10.1371/journal.pone.0092798) | PubMed: [24656130](https://pubmed.ncbi.nlm.nih.gov/24656130/) | [PDF](papers/Méric_2014.pdf)

- **Lefébure et al. (2010)** Evolutionary dynamics of complete Campylobacter pan-genomes and the bacterial species concept. *Genome Biology and Evolution*, 2:646-655. DOI: [10.1093/gbe/evq048](https://doi.org/10.1093/gbe/evq048) | PubMed: [20688752](https://pubmed.ncbi.nlm.nih.gov/20688752/) | [PDF](papers/Lefébure_2010.pdf)

- **Kim et al. (2022)** Comparative genomics of Campylobacter jejuni from clinical campylobacteriosis stool specimens. *Gut Pathogens*, 14:50. DOI: [10.1186/s13099-022-00520-1](https://doi.org/10.1186/s13099-022-00520-1) | PubMed: [36476389](https://pubmed.ncbi.nlm.nih.gov/36476389/) | [PDF](papers/Kim_2022.pdf)

- **Friis et al. (2010)** Genomic characterization of Campylobacter jejuni strain M1. *PLOS ONE*, 5(8):e12253. DOI: [10.1371/journal.pone.0012253](https://doi.org/10.1371/journal.pone.0012253) | PubMed: [20808928](https://pubmed.ncbi.nlm.nih.gov/20808928/) | [PDF](papers/Friis_2010.pdf)

## 2. Core/Accessory Genome & Population Structure

- **Pascoe et al. (2021)** Genome-wide insights into population structure and host specificity of Campylobacter jejuni. *Scientific Reports*, 11:11218. DOI: [10.1038/s41598-021-89683-6](https://doi.org/10.1038/s41598-021-89683-6) | PubMed: [33990625](https://pubmed.ncbi.nlm.nih.gov/33990625/) | [PDF](papers/Pascoe_2021.pdf)

- **Sheppard et al. (2014)** Cryptic ecology among host generalist Campylobacter jejuni in domestic animals. *Molecular Ecology*, 23(10):2442-2451. DOI: [10.1111/mec.12742](https://doi.org/10.1111/mec.12742) | PubMed: [24689900](https://pubmed.ncbi.nlm.nih.gov/24689900/) | [PDF](papers/Sheppard_2014.pdf)

- **Sheppard et al. (2013)** Genome-wide association study identifies vitamin B5 biosynthesis as a host specificity factor in Campylobacter. *PNAS*, 110(29):11923-11927. DOI: [10.1073/pnas.1305559110](https://doi.org/10.1073/pnas.1305559110) | PubMed: [23818615](https://pubmed.ncbi.nlm.nih.gov/23818615/) | [PDF](papers/Sheppard_2013.pdf)

- **Sheppard et al. (2018)** The evolution of Campylobacter jejuni and Campylobacter coli. In: *Campylobacter*, pp. 1-30. DOI: [10.1016/B978-0-12-803623-5.00001-6](https://doi.org/10.1016/B978-0-12-803623-5.00001-6)

- **Hepworth et al. (2011)** Genomic variations define divergence of water/wildlife-associated Campylobacter jejuni niche specialists from common clonal complexes. *Environmental Microbiology*, 13(6):1549-1560. DOI: [10.1111/j.1462-2920.2011.02461.x](https://doi.org/10.1111/j.1462-2920.2011.02461.x) | PubMed: [21418499](https://pubmed.ncbi.nlm.nih.gov/21418499/)

- **Yahara et al. (2017)** Genome-wide association of functional traits linked with Campylobacter jejuni survival from farm to fork. *Environmental Microbiology*, 19(1):361-380. DOI: [10.1111/1462-2920.13628](https://doi.org/10.1111/1462-2920.13628) | PubMed: [27883255](https://pubmed.ncbi.nlm.nih.gov/27883255/)

- **Baily et al. (2015)** Evidence of land-sea transfer of the zoonotic pathogen Campylobacter to a wildlife marine sentinel species. *Molecular Ecology*, 24(1):208-221. DOI: [10.1111/mec.13001](https://doi.org/10.1111/mec.13001) | PubMed: [25413890](https://pubmed.ncbi.nlm.nih.gov/25413890/)

- **Pascoe et al. (2015)** Enhanced biofilm formation and multi-host transmission evolve from divergent genetic backgrounds in Campylobacter jejuni. *Environmental Microbiology*, 17(11):4779-4789. DOI: [10.1111/1462-2920.13051](https://doi.org/10.1111/1462-2920.13051) | PubMed: [26373338](https://pubmed.ncbi.nlm.nih.gov/26373338/) | [PDF](papers/Pascoe_2015.pdf)

- **Kovanen et al. (2016)** Genomic insights from whole genome sequencing of four clonal outbreak Campylobacter jejuni assessed within the global C. jejuni population. *BMC Genomics*, 17:1-14. DOI: [10.1186/s12864-016-3340-8](https://doi.org/10.1186/s12864-016-3340-8) | PubMed: [28025473](https://pubmed.ncbi.nlm.nih.gov/28025473/) | [PDF](papers/Kovanen_2016.pdf)

- **Gripp et al. (2011)** Closely related Campylobacter jejuni strains from different sources reveal a generalist rather than a specialist lifestyle. *BMC Genomics*, 12:584. DOI: [10.1186/1471-2164-12-584](https://doi.org/10.1186/1471-2164-12-584) | PubMed: [22122991](https://pubmed.ncbi.nlm.nih.gov/22122991/) | [PDF](papers/Gripp_2011.pdf)

- **Skarp et al. (2015)** Comparative genomics and genome biology of invasive Campylobacter jejuni. *Scientific Reports*, 5:17300. DOI: [10.1038/srep17300](https://doi.org/10.1038/srep17300) | PubMed: [26603914](https://pubmed.ncbi.nlm.nih.gov/26603914/) | [PDF](papers/Skarp_2015.pdf)

- **Mourkas et al. (2020)** Agricultural intensification and the evolution of host specialism in the enteric pathogen Campylobacter jejuni. *PNAS*, 117(20):11018-11028. DOI: [10.1073/pnas.1917168117](https://doi.org/10.1073/pnas.1917168117) | PubMed: [32366649](https://pubmed.ncbi.nlm.nih.gov/32366649/) | [PDF](papers/Mourkas_2020.pdf)

- **Kivistö et al. (2014)** Evolution and comparative genomics of Campylobacter jejuni ST-677 clonal complex. *Genome Biology and Evolution*, 6(9):2424-2438. DOI: [10.1093/gbe/evu194](https://doi.org/10.1093/gbe/evu194) | PubMed: [25193305](https://pubmed.ncbi.nlm.nih.gov/25193305/) | [PDF](papers/Kivistö_2014.pdf)

## 3. Antimicrobial Resistance (AMR)

- **Wainaina et al. (2023)** Whole-genome-based characterization of Campylobacter jejuni from human patients with gastroenteritis collected over an 18 year period reveals increasing prevalence of antimicrobial resistance. *Microbial Genomics*, 9(3):mgen000941. DOI: [10.1099/mgen.0.000941](https://doi.org/10.1099/mgen.0.000941) | PubMed: [36947575](https://pubmed.ncbi.nlm.nih.gov/36947575/) | [PDF](papers/Wainaina_2023.pdf)

- **Elhadidy et al. (2020)** Prediction of antimicrobial resistance in clinical Campylobacter jejuni isolates from whole-genome sequencing data. *European Journal of Clinical Microbiology & Infectious Diseases*, 39:2265-2272. DOI: [10.1007/s10096-020-04043-y](https://doi.org/10.1007/s10096-020-04043-y) | PubMed: [32974772](https://pubmed.ncbi.nlm.nih.gov/32974772/) | [PDF](papers/Elhadidy_2020.pdf)

- **Zhao et al. (2021)** WGS-Based Prediction and Analysis of Antimicrobial Resistance in Campylobacter jejuni Isolates From Israel. *Frontiers in Cellular and Infection Microbiology*, 10:365. DOI: [10.3389/fcimb.2020.00365](https://doi.org/10.3389/fcimb.2020.00365) | PubMed: [32903472](https://pubmed.ncbi.nlm.nih.gov/32903472/) | [PDF](papers/Zhao_2021.pdf)

- **Crespo et al. (2022)** Genomic Analysis and Antimicrobial Resistance of Campylobacter jejuni and Campylobacter coli in Peru. *Frontiers in Microbiology*, 12:802404. DOI: [10.3389/fmicb.2021.802404](https://doi.org/10.3389/fmicb.2021.802404) | PubMed: [35087501](https://pubmed.ncbi.nlm.nih.gov/35087501/) | [PDF](papers/Crespo_2022.pdf)

- **Aksomaitiene et al. (2023)** Whole-Genome Sequencing-Based Profiling of Antimicrobial Resistance Genes and Core-Genome Multilocus Sequence Typing of Campylobacter jejuni from Different Sources in Lithuania. *Antibiotics*, 12(11):1579. DOI: [10.3390/antibiotics12111579](https://doi.org/10.3390/antibiotics12111579) | PubMed: [37958998](https://pubmed.ncbi.nlm.nih.gov/37958998/) | [PDF](papers/Aksomaitiene_2023.pdf)

- **Gupta et al. (2021)** Whole genome-based characterisation of antimicrobial resistance and genetic diversity in Campylobacter jejuni and Campylobacter coli from ruminants. *Scientific Reports*, 11:10200. DOI: [10.1038/s41598-021-88318-0](https://doi.org/10.1038/s41598-021-88318-0) | PubMed: [33986376](https://pubmed.ncbi.nlm.nih.gov/33986376/) | [PDF](papers/Gupta_2021.pdf)

- **Whitehouse et al. (2018)** Use of whole-genome sequencing for Campylobacter surveillance from NARMS retail poultry in the United States in 2015. *Food Microbiology*, 73:122-128. DOI: [10.1016/j.fm.2018.01.018](https://doi.org/10.1016/j.fm.2018.01.018) | PubMed: [29526190](https://pubmed.ncbi.nlm.nih.gov/29526190/) | [PDF](papers/Whitehouse_2018.pdf)

- **Talukdar et al. (2022)** The bile salt deoxycholate induces Campylobacter jejuni genetic point mutations that promote increased antibiotic resistance and fitness. *Frontiers in Microbiology*, 13:1062464. DOI: [10.3389/fmicb.2022.1062464](https://doi.org/10.3389/fmicb.2022.1062464) | PubMed: [36619995](https://pubmed.ncbi.nlm.nih.gov/36619995/) | [PDF](papers/Talukdar_2022.pdf)

## 4. Virulence & Pathogenicity

- **Burnham & Hendrixson (2018)** Campylobacter jejuni: collective components promoting a successful enteric lifestyle. *Nature Reviews Microbiology*, 16:551-565. DOI: [10.1038/s41579-018-0037-9](https://doi.org/10.1038/s41579-018-0037-9) | PubMed: [29892020](https://pubmed.ncbi.nlm.nih.gov/29892020/)

- **Young et al. (2007)** Campylobacter jejuni: molecular biology and pathogenesis. *Nature Reviews Microbiology*, 5:665-679. DOI: [10.1038/nrmicro1718](https://doi.org/10.1038/nrmicro1718) | PubMed: [17703225](https://pubmed.ncbi.nlm.nih.gov/17703225/)

- **Dasti et al. (2010)** Campylobacter jejuni: a brief overview on pathogenicity-associated factors and disease-mediating mechanisms. *International Journal of Medical Microbiology*, 300(4):205-211. DOI: [10.1016/j.ijmm.2009.07.002](https://doi.org/10.1016/j.ijmm.2009.07.002) | PubMed: [19665925](https://pubmed.ncbi.nlm.nih.gov/19665925/)

- **Bolton (2015)** Campylobacter virulence and survival factors. *Food Microbiology*, 48:99-108. DOI: [10.1016/j.fm.2014.11.017](https://doi.org/10.1016/j.fm.2014.11.017) | PubMed: [25790997](https://pubmed.ncbi.nlm.nih.gov/25790997/)

- **Liu et al. (2024)** Campylobacter jejuni virulence factors: update on emerging issues and trends. *Journal of Biomedical Science*, 31:91. DOI: [10.1186/s12929-024-01033-6](https://doi.org/10.1186/s12929-024-01033-6) | PubMed: [39223604](https://pubmed.ncbi.nlm.nih.gov/39223604/) | [PDF](papers/Liu_2024.pdf)

- **Peters et al. (2021)** Campylobacter jejuni genotypes are associated with post-infection irritable bowel syndrome in humans. *Communications Biology*, 4:1015. DOI: [10.1038/s42003-021-02554-8](https://doi.org/10.1038/s42003-021-02554-8) | PubMed: [34462533](https://pubmed.ncbi.nlm.nih.gov/34462533/) | [PDF](papers/Peters_2021.pdf)

- **Heikema et al. (2021)** Biomolecule sulphation and novel methylations related to Guillain-Barré syndrome-associated Campylobacter jejuni serotype HS:19. *Microbial Genomics*, 7(11):000660. DOI: [10.1099/mgen.0.000660](https://doi.org/10.1099/mgen.0.000660) | PubMed: [34723785](https://pubmed.ncbi.nlm.nih.gov/34723785/) | [PDF](papers/Heikema_2021.pdf)

## 5. MLST & Phylogenomics

- **Dingle et al. (2001)** Multilocus sequence typing system for Campylobacter jejuni. *Journal of Clinical Microbiology*, 39(1):14-23. DOI: [10.1128/JCM.39.1.14-23.2001](https://doi.org/10.1128/JCM.39.1.14-23.2001) | PubMed: [11136741](https://pubmed.ncbi.nlm.nih.gov/11136741/) | [PDF](papers/Dingle_2001.pdf)

- **Cody et al. (2017)** Core Genome Multilocus Sequence Typing Scheme for Stable, Comparative Analyses of Campylobacter jejuni and C. coli Human Disease Isolates. *Journal of Clinical Microbiology*, 55(7):2086-2097. DOI: [10.1128/JCM.00080-17](https://doi.org/10.1128/JCM.00080-17) | PubMed: [28446571](https://pubmed.ncbi.nlm.nih.gov/28446571/) | [PDF](papers/Cody_2017.pdf)

- **Mohan et al. (2023)** Evaluation of core genome and whole genome multilocus sequence typing schemes for Campylobacter jejuni and Campylobacter coli outbreak detection in the USA. *Journal of Clinical Microbiology*, 61(6):e00258-23. DOI: [10.1128/jcm.00258-23](https://doi.org/10.1128/jcm.00258-23) | PubMed: [37133905](https://pubmed.ncbi.nlm.nih.gov/37133905/) | [PDF](papers/Mohan_2023.pdf)

- **Jolley et al. (2018)** Open-access bacterial population genomics: BIGSdb software, the PubMLST.org website and their applications. *Wellcome Open Research*, 3:124. DOI: [10.12688/wellcomeopenres.14826.1](https://doi.org/10.12688/wellcomeopenres.14826.1) | PubMed: [30345391](https://pubmed.ncbi.nlm.nih.gov/30345391/) | [PDF](papers/Jolley_2018.pdf)

- **Llarena et al. (2017)** Whole-genome sequencing in epidemiology of Campylobacter jejuni infections. *Journal of Clinical Microbiology*, 55(5):1269-1275. DOI: [10.1128/JCM.00017-17](https://doi.org/10.1128/JCM.00017-17) | PubMed: [28228491](https://pubmed.ncbi.nlm.nih.gov/28228491/) | [PDF](papers/Llarena_2017.pdf)

## 6. Reference Genome (NCTC 11168)

- **Parkhill et al. (2000)** The genome sequence of the food-borne pathogen Campylobacter jejuni reveals hypervariable sequences. *Nature*, 403:665-668. DOI: [10.1038/35001088](https://doi.org/10.1038/35001088) | PubMed: [10688204](https://pubmed.ncbi.nlm.nih.gov/10688204/) | [PDF](papers/Parkhill_2000.pdf)

- **Gundogdu et al. (2007)** Re-annotation and re-analysis of the Campylobacter jejuni NCTC11168 genome sequence. *BMC Genomics*, 8:162. DOI: [10.1186/1471-2164-8-162](https://doi.org/10.1186/1471-2164-8-162) | PubMed: [17565669](https://pubmed.ncbi.nlm.nih.gov/17565669/) | [PDF](papers/Gundogdu_2007.pdf)

- **Pearson et al. (2012)** Complete Genome Sequence of a Variant of Campylobacter jejuni NCTC 11168. *Journal of Bacteriology*, 194(22):6298-6299. DOI: [10.1128/JB.01385-12](https://doi.org/10.1128/JB.01385-12) | PubMed: [23105049](https://pubmed.ncbi.nlm.nih.gov/23105049/) | [PDF](papers/Pearson_2012.pdf)

- **Revez et al. (2019)** Domestication of Campylobacter jejuni NCTC 11168. *Microbial Genomics*, 5(7):e000279. DOI: [10.1099/mgen.0.000279](https://doi.org/10.1099/mgen.0.000279) | PubMed: [31310201](https://pubmed.ncbi.nlm.nih.gov/31310201/) | [PDF](papers/Revez_2019.pdf)

## 7. Methodology References

### Pangenome Tools

- **Page et al. (2015)** Roary: rapid large-scale prokaryote pan genome analysis. *Bioinformatics*, 31(22):3691-3693. DOI: [10.1093/bioinformatics/btv421](https://doi.org/10.1093/bioinformatics/btv421) | PubMed: [26198102](https://pubmed.ncbi.nlm.nih.gov/26198102/) | [PDF](papers/Page_2015.pdf)

- **Tonkin-Hill et al. (2020)** Producing polished prokaryotic pangenomes with the Panaroo pipeline. *Genome Biology*, 21:180. DOI: [10.1186/s13059-020-02090-4](https://doi.org/10.1186/s13059-020-02090-4) | PubMed: [32698896](https://pubmed.ncbi.nlm.nih.gov/32698896/) | [PDF](papers/Tonkin-Hill_2020.pdf)

- **Fu et al. (2012)** CD-HIT: accelerated for clustering the next-generation sequencing data. *Bioinformatics*, 28(23):3150-3152. DOI: [10.1093/bioinformatics/bts565](https://doi.org/10.1093/bioinformatics/bts565) | PubMed: [23060610](https://pubmed.ncbi.nlm.nih.gov/23060610/) | [PDF](papers/Fu_2012.pdf)

### Genome Annotation & Workflow

- **Seemann (2014)** Prokka: rapid prokaryotic genome annotation. *Bioinformatics*, 30(14):2068-2069. DOI: [10.1093/bioinformatics/btu153](https://doi.org/10.1093/bioinformatics/btu153) | PubMed: [24642063](https://pubmed.ncbi.nlm.nih.gov/24642063/)

- **Mölder et al. (2021)** Sustainable data analysis with Snakemake. *F1000Research*, 10:33. DOI: [10.12688/f1000research.29032.3](https://doi.org/10.12688/f1000research.29032.3) | PubMed: [34035898](https://pubmed.ncbi.nlm.nih.gov/34035898/) | [PDF](papers/Mölder_2021.pdf)

### Genomic Distance & Quality

- **Ondov et al. (2016)** Mash: fast genome and metagenome distance estimation using MinHash. *Genome Biology*, 17:132. DOI: [10.1186/s13059-016-0997-x](https://doi.org/10.1186/s13059-016-0997-x) | PubMed: [27323842](https://pubmed.ncbi.nlm.nih.gov/27323842/) | [PDF](papers/Ondov_2016.pdf)

- **Schwengers et al. (2021)** Bakta: rapid and standardized annotation of bacterial genomes via alignment-free sequence identification. *Microbial Genomics*, 7(11):000685. DOI: [10.1099/mgen.0.000685](https://doi.org/10.1099/mgen.0.000685) | PubMed: [34739369](https://pubmed.ncbi.nlm.nih.gov/34739369/) | [PDF](papers/Schwengers_2021.pdf)

- **Parks et al. (2015)** CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. *Genome Research*, 25:1043-1055. DOI: [10.1101/gr.186072.114](https://doi.org/10.1101/gr.186072.114) | PubMed: [25977477](https://pubmed.ncbi.nlm.nih.gov/25977477/) | [PDF](papers/Parks_2015.pdf)

### Dimensionality Reduction & Clustering

- **McInnes et al. (2018)** UMAP: Uniform Manifold Approximation and Projection for Dimension Reduction. *arXiv*:1802.03426. Link: [arXiv:1802.03426](https://arxiv.org/abs/1802.03426) | [PDF](papers/McInnes_2018.pdf)

- **McInnes et al. (2017)** hdbscan: Hierarchical density based clustering. *Journal of Open Source Software*, 2(11):205. DOI: [10.21105/joss.00205](https://doi.org/10.21105/joss.00205) | [PDF](papers/McInnes_2017.pdf)

### NMF & Pangenome Matrix Decomposition

- **Chauhan et al. (2025)** Decomposition of the pangenome matrix reveals a structure in gene distribution in the Escherichia coli species. *mSphere*, 10(1):e00532-24. DOI: [10.1128/msphere.00532-24](https://doi.org/10.1128/msphere.00532-24) | PubMed: [39745367](https://pubmed.ncbi.nlm.nih.gov/39745367/) | [PDF](papers/Chauhan_2025.pdf)

- **Devarenne et al. (2008)** Nonnegative Matrix Factorization: An Analytical and Interpretive Tool in Computational Biology. *PLOS Computational Biology*, 4(7):e1000029. DOI: [10.1371/journal.pcbi.1000029](https://doi.org/10.1371/journal.pcbi.1000029) | PubMed: [18654623](https://pubmed.ncbi.nlm.nih.gov/18654623/) | [PDF](papers/Devarenne_2008.pdf)

## 8. BioProject-Associated Papers

Papers and metadata linked to the 96 valid BioProjects represented in our 493-strain dataset (from `output/1a_genome_metadata.csv`). An additional ~35 rows have data-quality issues (assembly method names in the BioProject column).

### Major BioProjects (≥10 strains)

#### PRJNA1087358 — Jeonbuk National University (South Korea) · 73 strains

*Complete genome sequences of C. jejuni isolated from humans, chickens, and dogs (102 assemblies)*

No dedicated publication found — direct NCBI submission.

#### PRJNA268846 — University of Helsinki · 55 strains

*Invasive C. jejuni comparative genomics*

- **Skarp et al. (2015)** Comparative genomics and genome biology of invasive Campylobacter jejuni. *Scientific Reports*, 5:17300. DOI: [10.1038/srep17300](https://doi.org/10.1038/srep17300) | PubMed: [26603914](https://pubmed.ncbi.nlm.nih.gov/26603914/) *(also in §2)* | [PDF](papers/Skarp_2015.pdf)

- **Kivistö et al. (2014)** Evolution and comparative genomics of Campylobacter jejuni ST-677 clonal complex. *Genome Biology and Evolution*, 6(9):2424-2438. DOI: [10.1093/gbe/evu194](https://doi.org/10.1093/gbe/evu194) | PubMed: [25193305](https://pubmed.ncbi.nlm.nih.gov/25193305/) *(also in §2)* | [PDF](papers/Kivistö_2014.pdf)

#### PRJNA344714 — University of Helsinki · 50 strains

*Genome analysis of invasive C. jejuni with emphasis on generalist lineage ST45CC (31 complete assemblies, blood and fecal isolates)*

No dedicated publication found. The BioProject was registered 2018-08-09 but no linked manuscript has been identified. Related work from the same group uses PRJNA268846.

#### PRJDB15932 — Hiroshima University / NICED Kolkata · 46 strains

*Genomic surveillance of Campylobacter isolates from diarrhea patients in Kolkata, India*

No dedicated publication found — direct NCBI submission. Funded by AMED grant JP22wm0225012.

#### PRJNA1114755 — USDA · 23 strains

*Whole genome sequencing of Campylobacter food isolates*

No dedicated publication found — direct NCBI submission.

#### PRJNA812595 — Utah Public Health Laboratory · 16 strains

*UPHL Gram-negative bacteria — public health pathogen surveillance*

No dedicated publication found — direct NCBI submission.

#### PRJNA239251 — CDC · 15 strains

*CDC surveillance Campylobacter genomes*

No dedicated publication found — direct NCBI submission (CDC surveillance).

#### PRJNA776013 — Naval Medical Research Center · 14 strains

*Campylobacter jejuni from Guillain-Barré syndrome cases in Peru*

- **Quino et al. (2022)** Emergence and molecular epidemiology of Campylobacter jejuni ST-2993 associated with a large outbreak of Guillain-Barré syndrome in Peru. *Microbiology Spectrum*, 10(5):e01187-22. DOI: [10.1128/spectrum.01187-22](https://doi.org/10.1128/spectrum.01187-22) | PubMed: [35972275](https://pubmed.ncbi.nlm.nih.gov/35972275/) | [PDF](papers/Quino_2022.pdf)

- **Heikema et al. (2021)** Biomolecule sulphation and novel methylations related to Guillain-Barré syndrome-associated Campylobacter jejuni serotype HS:19. *Microbial Genomics*, 7(11):000660. DOI: [10.1099/mgen.0.000660](https://doi.org/10.1099/mgen.0.000660) | PubMed: [34723785](https://pubmed.ncbi.nlm.nih.gov/34723785/) *(also in §4)* | [PDF](papers/Heikema_2021.pdf)

#### PRJNA634604 — USDA ARS · 13 strains

*Campylobacter jejuni genomes for AMR and virulence studies*

- **Talukdar et al. (2022)** The bile salt deoxycholate induces Campylobacter jejuni genetic point mutations that promote increased antibiotic resistance and fitness. *Frontiers in Microbiology*, 13:1062464. DOI: [10.3389/fmicb.2022.1062464](https://doi.org/10.3389/fmicb.2022.1062464) | PubMed: [36619995](https://pubmed.ncbi.nlm.nih.gov/36619995/) *(also in §3)* | [PDF](papers/Talukdar_2022.pdf)

- **Heikema et al. (2021)** *(see PRJNA776013 above)*

#### PRJNA649348 — Henan Agricultural University · 10 strains

*C. jejuni efflux pump and multidrug resistance study*

No dedicated publication found — direct NCBI submission.

### Medium BioProjects (3–9 strains)

| BioProject | Strains | Submitter / Notes |
|-----------|---------|-------------------|
| PRJNA655459 | 8 | Mississippi State Univ — retail chicken / broiler farms |
| PRJNA971184 | 7 | *(no metadata retrieved)* |
| PRJNA233954 | 6 | *(no metadata retrieved)* |
| PRJEB60038 | 6 | *(European project)* |
| PRJNA637789 | 5 | *(no metadata retrieved)* |
| PRJNA562719 | 5 | CDC — FDA-CDC Antimicrobial Resistance Bank |
| PRJNA215043 | 5 | *(no metadata retrieved)* |
| PRJNA643291 | 4 | *(no metadata retrieved)* |
| PRJNA520992 | 4 | *(no metadata retrieved)* |
| PRJNA854369 | 3 | *(no metadata retrieved)* |
| PRJNA641382 | 3 | *(no metadata retrieved)* |
| PRJNA276739 | 3 | *(no metadata retrieved)* |
| PRJNA231221 | 3 | *(no metadata retrieved)* |
| PRJEB6403 | 3 | *(European project)* |

### Small BioProjects (1–2 strains)

68 additional BioProjects each contribute 1–2 strains (total ~80 strains). These include:

| BioProject | Strains | Notes |
|-----------|---------|-------|
| PRJNA168989 | 1 | Univ Nottingham — *C. jejuni* PT14 (Brathwaite 2013, Hooton 2014) |
| PRJNA1087001 | 1 | Univ Melbourne — Nanopore variant calling (Hall 2024) |
| PRJNA1101017 | 1 | FDA CFSAN — GenomeTrakr validation |
| PRJNA1078568 | 1 | Southwest Medical Univ (China) — MDR blood isolate |
| PRJNA1224609 | 1 | Vircell S.L. (Spain) — diagnostic assay design |
| PRJNA492384 | 1 | China Agricultural Univ — erm(B) resistance (Wang 2022 related) |
| PRJNA634324 | 1 | Univ Utrecht — human pathogenic strain 129108 |
| PRJNA175 | 1 | TIGR legacy — likely BV-BRC metadata error (*Prevotella intermedia*) |
| *(60 others)* | 1–2 | Various surveillance, clinical, and research submissions |

### Data-Quality Note

~35 rows in the dataset have assembly method names (e.g., "Unicycler v. v0.3.4", "Canu v. v2.2") in the `bioproject_accession` column instead of valid accessions. These represent BV-BRC metadata issues where the assembly method field was shifted into the BioProject column. The genomes themselves passed all QC filters and are retained in the dataset.

---

*Last updated: 2026-02-17 | Generated for the pyphylon C. jejuni pangenome analysis pipeline*
