{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 1a. Filter for high-quality genomes to download\n",
    "\n",
    "In this notebook, we will use __`pyphylon`__'s `download` and `qcqa` modules to select candidate genomes to download for pangenome generation.\n",
    "\n",
    "In this example we will select genomes for download from [BV-BRC](https://www.bv-brc.org/)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Setup"
   ]
  },
  {
   "cell_type": "code",
   "metadata": {},
   "source": [
    "import pandas as pd\n",
    "import plotly.express as px\n",
    "import seaborn as sns\n",
    "import matplotlib\n",
    "from matplotlib import pyplot as plt\n",
    "\n",
    "from pyphylon.downloads import get_scaffold_n50_for_species, query_bvbrc_genomes\n",
    "from pyphylon.util import load_config\n",
    "import pyphylon.qcqa as qcqa\n",
    "import os\n",
    "# read config.yml to make sure reuse_temp is respected\n",
    "import yaml\n",
    "reuse_temp = False\n",
    "temp_folder = os.path.join(\"../temp/\")\n",
    "\n",
    "# Temp override from config.yml\n",
    "with open(\"config.yml\", 'r') as f:\n",
    "    config = yaml.safe_load(f)\n",
    "if config[\"REUSE_TEMP\"]:\n",
    "    reuse_temp = True\n",
    "if config[\"REUSE_TEMP_DIR\"]:\n",
    "    temp_folder = config[\"REUSE_TEMP_DIR\"]\n",
    "\n",
    "print(f\"Temp folder: {temp_folder}\")\n",
    "\n",
    "# Print work directory\n",
    "print(f\"Working directory: {os.getcwd()}\")\n",
    "\n",
    "# working directory is one level into the repository\n",
    "input_folder = os.path.join(\"../input/\")\n",
    "print(f\"Input folder: {input_folder}\")\n",
    "\n",
    "\n",
    "\n",
    "output_folder = os.path.join(\"../output/\")\n",
    "print(f\"Output folder: {output_folder}\")"
   ],
   "outputs": [],
   "execution_count": null
  },
  {
   "cell_type": "code",
   "metadata": {},
   "source": [
    "plt.rcParams[\"figure.dpi\"] = 200\n",
    "sns.set_palette(\"deep\")\n",
    "sns.set_context(\"paper\")\n",
    "sns.set_style(\"whitegrid\")\n"
   ],
   "outputs": [],
   "execution_count": null
  },
  {
   "cell_type": "code",
   "metadata": {},
   "source": "CONFIG = load_config(\"config.yml\")\nWORKDIR = CONFIG[\"WORKDIR\"]\nSPECIES_NAME = CONFIG[\"SPECIES_NAME\"]\nTAXON_ID = CONFIG[\"TAXON_ID\"]\nDEBUG = CONFIG[\"DEBUG\"]",
   "outputs": [],
   "execution_count": null
  },
  {
   "cell_type": "code",
   "metadata": {},
   "source": "# Query BV-BRC API for Complete + Good quality genomes\ngenome_df = query_bvbrc_genomes(TAXON_ID, genome_status='Complete', genome_quality='Good')\ngenome_df = genome_df.set_index('genome_id')\nsummary = genome_df.copy()\nmetadata = genome_df.copy()\nprint(f\"Retrieved {summary.shape[0]} genomes from BV-BRC API\")\nsummary.shape",
   "outputs": [],
   "execution_count": null
  },
  {
   "cell_type": "code",
   "metadata": {},
   "source": [
    "summary.genome_name.str.contains(SPECIES_NAME).sum()"
   ],
   "outputs": [],
   "execution_count": null
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Filter metadata for species of interest"
   ]
  },
  {
   "cell_type": "code",
   "metadata": {},
   "source": [
    "# How many strains of the species/genus are available\n",
    "species_summary = qcqa.filter_by_species(summary, CONFIG['SPECIES_NAME'])\n",
    "metadata_summary = qcqa.filter_by_species(metadata, CONFIG['SPECIES_NAME'])\n",
    "\n",
    "display(\n",
    "    species_summary.shape,\n",
    "    species_summary.head()\n",
    ")\n"
   ],
   "outputs": [],
   "execution_count": null
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Plot unfiltered dataset"
   ]
  },
  {
   "cell_type": "code",
   "metadata": {},
   "source": [
    "# Find the scaffold N50 score of the reference genome for the organism of interest\n",
    "# Either visit the NCBI website or retrieve it using the following method (~20 seconds)\n",
    "scaffold_n50 = get_scaffold_n50_for_species(species_summary.taxon_id.mode().values[0])\n",
    "scaffold_n50"
   ],
   "outputs": [],
   "execution_count": null
  },
  {
   "cell_type": "code",
   "metadata": {},
   "source": [
    "# Initial unfiltered strain plot\n",
    "h = sns.jointplot(\n",
    "    data=species_summary,\n",
    "    x=\"genome_length\",\n",
    "    y=\"patric_cds\",\n",
    "    hue=\"genome_status\",\n",
    "    alpha=0.75,\n",
    "    height=4\n",
    ")\n",
    "\n",
    "h.ax_joint.legend(\n",
    "    title='BV-BRC\\nstrain type',\n",
    ")\n",
    "\n",
    "h.ax_joint.set_xlabel(\"genome length\")\n",
    "h.ax_joint.set_ylabel(\"BV-BRC predicted gene count\")\n",
    "plt.savefig(output_folder + '1a_unfiltered_strains.png', bbox_inches='tight')\n",
    "plt.show()\n"
   ],
   "outputs": [],
   "execution_count": null
  },
  {
   "cell_type": "code",
   "metadata": {},
   "source": [
    "# Find reference strain N50 value from NCBI Genome and multiply by 0.85\n",
    "# If your species/genus has multiple reference strains, pick the smallest by genome length\n",
    "# C. jejuni reference (NCTC 11168) is a single-chromosome ~1.64 Mb genome\n",
    "# The 0.85 multiplier gives a threshold of ~1.39 Mb\n",
    "\n",
    "# Only applies for Complete sequences\n",
    "species_complete_summary = species_summary[species_summary.genome_status == 'Complete']\n",
    "\n",
    "fig, ax = plt.subplots()\n",
    "\n",
    "# Set threshold as 0.85 * Scaffold N50 score\n",
    "species_ref_n50 = scaffold_n50\n",
    "min_thresh_n50 = int(0.85 * species_ref_n50)\n",
    "\n",
    "# Most (if not all) Complete sequences pass this threshold\n",
    "sns.histplot(species_complete_summary.contig_n50.dropna().astype('int'), ax=ax)\n",
    "plt.axvline(x=min_thresh_n50, color='#ff00ff', linestyle='--')\n",
    "plt.savefig(output_folder + '1a_unfiltered_n50.png', bbox_inches='tight')"
   ],
   "outputs": [],
   "execution_count": null
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Initial Filtration Report"
   ]
  },
  {
   "cell_type": "code",
   "metadata": {},
   "source": "# Complete sequences get filtered by their N50 and L50 scores\n# Other WGS sequences get filtered by their contig count\n# CheckM contamination & completeness filtering uses explicit cutoffs\n# from Parks et al. (2015), Genome Research 25:1043-1055\nfiltered_species_summary, df_filtration = qcqa.filter_by_genome_quality(\n    species_summary,\n    min_thresh_n50=min_thresh_n50,\n    max_contig=None,\n    contamination_cutoff=5.0,       # explicit: keep genomes with <5% contamination\n    completeness_cutoff=90.0,       # explicit: keep genomes with >90% completeness\n    checkm_filter_statuses=('Complete', 'WGS'),\n    checkm_missing='keep',          # retain genomes without CheckM data\n    return_stats=True,\n)\n\ndisplay(\n    f'Filtered Strains:',\n    filtered_species_summary.shape,\n    f'------------------------------',\n    f'Filtration Report',\n    df_filtration\n)",
   "outputs": [],
   "execution_count": null
  },
  {
   "cell_type": "code",
   "metadata": {},
   "source": [
    "# Same initial plot but with only (first-pass) filtered strains\n",
    "# Inspect the distribution below\n",
    "# If WGS sequences don't form a tight cluster, consider additional manual filtering\n",
    "\n",
    "h = sns.jointplot(\n",
    "    data=filtered_species_summary,\n",
    "    x=\"genome_length\",\n",
    "    y=\"patric_cds\",\n",
    "    hue=\"genome_status\",\n",
    "    alpha=0.75,\n",
    "    height=4\n",
    ")\n",
    "\n",
    "h.ax_joint.legend(\n",
    " title='BV-BRC\\nstrain type'\n",
    ")\n",
    "\n",
    "h.ax_joint.set_xlabel(\"genome length\")\n",
    "h.ax_joint.set_ylabel(\"BV-BRC predicted gene count\")\n",
    "plt.savefig(output_folder + '1a_filtered_strains.png', bbox_inches='tight')\n",
    "plt.show()\n"
   ],
   "outputs": [],
   "execution_count": null
  },
  {
   "cell_type": "code",
   "metadata": {},
   "source": "# Ensure GC content makes sense\n# Remove any big outliers if present\n\nh = sns.jointplot(\n    data=filtered_species_summary,\n    x=\"gc_content\",\n    y=\"contigs\",\n    hue=\"genome_status\",\n    alpha=0.75,\n    height=4\n)\n\nh.ax_joint.legend(\n    title='BV-BRC\\nstrain type',\n    bbox_to_anchor=(1.45,1.4)\n)\n\nh.ax_joint.set_xlabel(\"GC Content\")\nh.ax_joint.set_ylabel(\"number of contigs\")\nplt.show()",
   "outputs": [],
   "execution_count": null
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Save (first-pass) filtered genome info files for download"
   ]
  },
  {
   "cell_type": "code",
   "metadata": {},
   "source": [
    "if DEBUG:\n",
    "    filtered_species_summary = filtered_species_summary[:50]\n",
    "    # filtered_species_metadata = filtered_species_metadata.loc[species_summary[:10].index,:]"
   ],
   "outputs": [],
   "execution_count": null
  },
  {
   "cell_type": "code",
   "metadata": {},
   "source": [
    "filtered_species_metadata = metadata.loc[filtered_species_summary.index]\n",
    "filtered_species_metadata"
   ],
   "outputs": [],
   "execution_count": null
  },
  {
   "cell_type": "code",
   "metadata": {},
   "source": "filtered_species_summary.to_csv(os.path.join(temp_folder, '1a_genome_summary.csv'))\nfiltered_species_metadata.to_csv(os.path.join(temp_folder,'1a_genome_metadata.csv'))",
   "outputs": [],
   "execution_count": null
  },
  {
   "cell_type": "code",
   "metadata": {},
   "source": "df_filtration.to_csv(os.path.join(output_folder, '1a_df_filtration.csv'))",
   "outputs": [],
   "execution_count": null
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "pyphylon",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.11"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
